test/unit/points/domain/test-tiers.js

```diff
@@ -0,0 +1,35 @@
+const expect = require('chai').expect
+const Tiers = require('../../../../app/points/domain/tiers')
+const pointsHelper = require('../../../helpers/points-helper')
+const Locations = require('../../../../app/staging/constants/locations')
+
+describe('points/domain/Tiers', function () {
+  it('throws an error when any property is undefined', function () {
+    expect(function () {
+      new Tiers(Locations.COMMUNITY,
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      undefined,
```

Comment:
might be worth doing this for all the arguments. A lot of code I know, but keeps all the bases tested in case any change or some other change impacts a different one.